grails-app/conf/Config.groovy

```diff
@@ -907,6 +907,12 @@ openboxes {
                 endpoint = "/${appName}/api/dashboard/openPurchaseOrdersCount"
             }
         }
+//        helpscout {
```

Comment:
Since this won't work like that you can get rid of this part.

Reply:
Or the HelpScout config could be an additional config since we will probably pull these values via API